### PR TITLE
Added the Project.md file

### PR DESCRIPTION
**Pull request by ymaheshwari1**
Added a basic Project description and project structure in the Project.md file

### CODE REVIEW

File: src/Project.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Project Documentation
+This project aims to build a webextension for Firefox browser, that can tag other Web extension in the browser.
```
Comment (Ajitesh13) [inline src/Project.md:2]: what does, "that can tag other web-extension" mean ?  you can either use the word "track" or any appropriate word in your convenience

**Comment by Ajitesh13** [comment]
Also, for passing the checks, please have a look over this comment: https://github.com/HakinCodes/extension-activity-logger/pull/10#issue-469242726

File: src/Project.md (hunk 1 above)
Comment (ymaheshwari1) [reply src/Project.md:2]: Actually I have copied this from the README.md file. But I will update it.

Comment (Ajitesh13) [reply src/Project.md:2]: So sorry, its my mistake then. I have fixed the typo in the readme. Thanks for pointing it out

**Pull request closed without merging**